src/reports_api/resources/milestone.py

```diff
@@ -24,12 +24,12 @@
 
 
 @cors_preflight('GET')
-@API.route('/<int:phase_id>', methods=['GET', 'OPTIONS'])
+@API.route('/phase/<int:phase_id>', methods=['GET', 'OPTIONS'])
```

Comment:
/phases/<int:phase_id> would be more appropriate I think. Let me know what do you think.

Reply:
updated